dodo/pgp_util.py

```diff
@@ -0,0 +1,111 @@
+#     Dodo - A graphical, hackable email client based on notmuch
+#     Copyright (C) 2021 - Aleks Kissinger
+#
+# This file is part of Dodo
+#
+# Dodo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Dodo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Dodo. If not, see <https://www.gnu.org/licenses/>.
+
+import email
+import email.utils
+import re
+from . import settings
+
+# gnupg is only needed for pgp/mime support, do not throw when not present
+try:
+    import gnupg
+except ImportError:
+    pass
+
+Gpg = gnupg.GPG(gnupghome=settings.gnupg_home, use_agent=True)
```

Comment:
Should this be inside the try block above?

Reply:
yes, indeed. Tx!